processor_function/README.md

```diff
@@ -0,0 +1,185 @@
+# Processor function
+
+<!-- markdownlint-disable-next-line MD013 -->
+The processor function (sometimes referred to as resource or deployment processor) uses [Azure Container Instances (ACI)](https://docs.microsoft.com/en-us/azure/container-instances/) together with the [CNAB container image](../CNAB_container/Dockerfile) based on [Azure CNAB Driver](https://github.com/deislabs/cnab-azure-driver) to execute workspace and workspace service deployments. The workspaces and workspace services are [Porter](https://porter.sh/) bundles.
+
+<!-- markdownlint-disable-next-line MD013 -->
+The processor function waits for Service Bus messages in the resource request queue containing the bundle details and the action to execute, prepares the environment for the execution in the ACI and follows through the deployment process. The deployment status is reported back to [the management API](../management_api_app/) using Service Bus messages but with in a deployment status update queue.
+
+## Prerequisites
+
+Tools:
+
+* [Python 3.x](https://www.python.org/downloads/)
```

Comment:
@sachinkundu I think we are using a specific version, right?

Reply:
The dev container has 3.8. Updating the README accordingly.